README.md

```diff
@@ -127,11 +98,7 @@ Their detailed documentation is available in their respective source locations:
 
 - [.NET](https://github.com/signalfx/signalfx-dotnet-tracing)
 - [Go](https://github.com/signalfx/signalfx-go-tracing)
-- [Java](https://github.com/signalfx/signalfx-java-tracing)
-- [Node.js](https://github.com/signalfx/signalfx-nodejs-tracing)
 - [PHP](https://github.com/signalfx/signalfx-php-tracing)
-- [Python](https://github.com/signalfx/signalfx-python-tracing)
-- [Ruby](https://github.com/signalfx/signalfx-ruby-tracing)
```

Comment:
We don't have Otel Ruby yet

Reply:
There was not `signalfx-ruby-tracing` folder even before this PR :D

Reply:
@owais will you approve now or should I do something about this? :)